python/foxglove-sdk/python/foxglove/_foxglove_py/websocket.pyi

```diff
@@ -115,9 +120,30 @@ class Parameter:
         self,
         name: str,
         *,
-        type: Optional["ParameterType"] = None,
-        value: Optional["AnyParameterValue"] = None,
+        value: Optional["AnyNativeParameterValue"] = None,
     ) -> None: ...
+    @staticmethod
+    def raw(
+        name: str,
+        *,
+        type: Optional["ParameterType"],
```

Comment:
Seems like this should have a default value of `None`. Probably `value` should too, since it's optional, though I'm not sure what use cases this serves; maybe forcing `value=None` is appropriate.